test/bats/azure.bats

```diff
@@ -125,13 +126,70 @@ setup() {
   [[ "${result//$'\r'}" -eq "${SECRET_VALUE}" ]]
 
   result=$(kubectl get secret foosecret -o json | jq '.metadata.ownerReferences | length')
-  [[ "$result" -eq "2" ]]
+  [[ "$result" == "2" ]]
```

Comment:
i think `==` fails for some systems. can we make sure we consistently use `-eq` in case people need to run this locally?